phpcs.xml.dist

```diff
@@ -95,17 +95,25 @@
     >tests/phpunit/integration/includes/REST_Setup.php</exclude-pattern>
   </rule>
 
-  <rule
-    ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint"
-  >
+  <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint">
     <properties>
       <property name="enableStaticTypeHint" value="false" />
       <property name="enableMixedTypeHint" value="false" />
       <property name="enableUnionTypeHint" value="false" />
     </properties>
+    <exclude
+      name="SlevomatCodingStandard.TypeHints.ReturnTypeHint.UselessAnnotation"
+    />
+    <exclude
+      name="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingTraversableTypeHintSpecification"
+    />
+
+    <!-- The main plugin file should be parsable by PHP < 7.0. -->
```

Comment:
```suggestion
    <!-- The main plugin file should be parsable by PHP < 7.2. -->
```

Reply:
For context, this comment is just copied from above.